LocalStorage/LocalStorageHelpers/RealmStorageHelper.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  RealmStorageHelper.swift
+//  WeatherDemoApp
+//
+//  Created by ahmed on 05/04/2024.
+//
+
+import RealmSwift
+import Combine
+
+final class RealmStorageHelper: LocalStorageProtocol {
+    private var realm = try! Realm()
```

Comment:
Force unwrapping is not allowed and should be handled with error handling and reporting.